lib/_emerge/actions.py

```diff
@@ -2634,13 +2635,45 @@ def apply_priorities(settings):
 	nice(settings)
 
 def nice(settings):
+	nice_value: str = settings.get("PORTAGE_NICENESS", "0")
```

Comment:
There was a version of these changes that did nothing unless `PORTAGE_NICENESS` was explicitly set in the environment. We need that version to avoid failures on prefix with old kernels. This version will always try to write to `/proc/self/autogroup` and that will fail on old kernels (even as root, simply doing `echo 10 > /proc/self/autogroup` fails on 2.6.32, which is still unfortunately in use in very old systems).

Reply:
Right, the somehow the improved version got lost: https://github.com/gentoo/portage/commit/055abe523c2c3f6c8f1dccfb53565209222f90c1#diff-dbd67931d37002a46abae37da609fef65a83cf1c90fe57325b2f73e0db831664R2638